docs/devel/e2e-tests.md

```diff
@@ -102,6 +102,18 @@ As mentioned earlier there are a host of other options that are available, but a
 - `rm -rf /var/run/kubernetes`, clear kube generated credentials, sometimes stale permissions can cause problems.
 - `sudo iptables -F`, clear ip tables rules left by the kube-proxy.
 
+## Kinds of tests
+
+We are working on implementing clearer partitioning of our e2e tests to make running a known set of tests easier (#10548).  Tests can be labeled with any of the following labels, in order of increasing precedence (that is, each label listed below supersedes the previous ones):
+
+- If a test has no labels, it is expected to run fast (under five minutes), be able to be run in parallel, and be consistent.
```

Comment:
We actually have quite a few tests that run > 2 minutes:

http://kubekins.dls.corp.google.com/job/kubernetes-e2e-gce/10331/testReport/(root)/Kubernetes%20e2e%20suite/

Part of the advantage of moving to a clearer partition is that we can run those 150 tests (everything that's not `[Serial]` or `[Disruptive]`) in parallel by default, and shove only the necessary tests to a serial run.  (That change is a WIP.)